internal/server/singleprocess/poll.go

```diff
@@ -158,16 +166,31 @@ func (s *service) runPollQueuer(
 			continue
 		}
 
-		resp, err := s.QueueJob(ctx, queueJobRequest)
-		if err != nil {
-			log.Warn("error queueing a poll job", "err", err)
-			time.Sleep(1 * time.Second)
+		totalRequests := len(queueJobRequests)
+		log.Trace("queueing jobs for poller", "job_total", totalRequests)
+
+		// If one job fails to queue, we break out of the entire queue loop
```

Comment:
I think important to note: but we _keep_ the ones that did succeed queueing, so its possible by design to have partial queueing here.

If we don't want that, its certainly possible to make the state store treat the entire thing like a transaction.